doc/source/billinge-group-standards.rst

```diff
@@ -0,0 +1,251 @@
+
+:tocdepth: -1
+
+.. index:: billinge-group-standards
+
+.. _billinge-group-standards:
+
+========================
+Billinge Group standards
+========================
+
+.. _github-commit-issue-practice:
+
```

Comment:
please can we have a few words here introducing this. Something like this:
"We present here standards of coding and coding workflows that have been adopted in the Billinge research group over multiple years to make our code more uniform and maintainable by a diverse and shifting team of students and post-docs.  They are mostly adopting what we have learned to be standards in the Python community, with some lessons learned along the way.  This is not the only way to do things, but since it has taken us lots of trial and error to develop them over time, we share them here in case they are useful to you."

Reply:
Yes, awesome phrase. Adopted.